Makefile

```diff
@@ -639,7 +638,6 @@ SPEED_OPTIMISED_SRC := $(SPEED_OPTIMISED_SRC) \
             drivers/rx_nrf24l01.c \
             drivers/rx_spi.c \
             drivers/rx_xn297.c \
-            drivers/pwm_output.c \
```

Comment:
I don't think it is worth going to that trouble for this smallish file. That would also make the code more convoluted. So in this case, I'd say, despite my earlier comments, it's probably best just to use a pragma.